File: js/30s/string/byteSize.js
/*Returns the length of a string in bytes.

Convert a given string to a Blob Object and find its size.*/

const byteSize = str => new Blob([str]).size

byteSize('😀'); // 4
byteSize('Hello World'); // 11
console.log(byteSize("ヾ(•ω•`)o"))

File: python/30s/string/byte_size.py
"""
Returns the length of a string in bytes.

utf-8 encodes a given string and find its length.
"""


def byte_size(string):
    return len(string.encode('utf-8'))


a = byte_size('😀')  # 4
b = byte_size('Hello World')  # 11
print(a, b)
